Loop/src/loop.hxx

```diff
@@ -517,6 +521,110 @@ public:
     } // for rank
   }
 
+  // Alex Dima:
```

Comment:
Don't put your name there. If your name is there, then others will be afraid of changing the code because they will defer to your authority. To find out who wrote what code one can use `git blame`.

Reply:
That is a good point, I  removed it from the comment.